packit_service/worker/events/copr.py

```diff
@@ -133,10 +133,14 @@ def pre_check(self):
 
         return True
 
+    def get_non_serializable_attributes(self):
+        skip = super().get_non_serializable_attributes()
+        skip.append("build")
+        return skip
```

Comment:
How about this?

```suggestion
        return super().get_non_serializable_attributes() + ["build"]
```

Reply:
Makes a new copy, not that that it would matter that much.

Reply:
And probably it is better to work on a new copy. So +1 for copying it. Thanks Nikola.